EIPS/eip-2502.md

```diff
@@ -0,0 +1,462 @@
+---
+eip: 2502
+title: µTopic Delegation
+author: Ricardo Guilherme Schmidt (@3esmit)
+discussions-to: https://ethereum-magicians.org/t/eip-2502-topic-delegation/3974
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-02-02
+requires: 2470
+
```

Comment:
```suggestiont
```
I *think* this is blocking CI.  Sadly, the error isn't particularly great as it just says "malformed field" but I *think* it is referring to this blank line.

Reply:
I think you're right. I really need to update the validator to propagate line numbers.